Formula/redo.rb

```diff
@@ -11,15 +13,35 @@ class Redo < Formula
     sha256 "630ac52a05e6b4586f1f78219bc9bce9d17cd67ab1b4acccb3d62256c2839bce" => :sierra
   end
 
-  resource "docs" do
-    url "https://github.com/apenwarr/redo.git", :branch => "man"
+  depends_on "python@2"
```

Comment:
We should be building against python3 if possible.

Reply:
It looks like this *may* be possible in the next release. I'm dismissing this for now.